### PR TITLE
Fixes #520: Py3 compatibility and printing username and ipAddress

### PR DESCRIPTION
**Pull request by jm66**

### CODE REVIEW

**Comment by kumahesh** [comment]
Works good in both version of python

File: samples/sessions_list.py [hunk 1]
```diff
@@ -66,22 +66,28 @@
...
-                          port=int(args.port))
+si = SmartConnectNoSSL(
+        host=args.host,
+        user=args.user,
+        pwd=password,
+        port=int(args.port)
+    )
 
-print "logged in to %s" % args.host
+print("logged in to %s" % args.host)
```
Comment (pgbidkar) [inline samples/sessions_list.py]: Hi @jm66 Thanks for making this change. Please can you make quick change, replace %s substitution with  string.format() method instead.

**Comment by pgbidkar** [comment]
> Works good in both version of python

Thanks Mahesh

File: samples/sessions_list.py (hunk 1 above)
Comment (jm66) [reply samples/sessions_list.py]: Will do. Thanks @pgbidkar

**Comment by jm66** [comment]
Should be good to go @pgbidkar

**Pull request merged**